interceptors/1.2/apidocs/allclasses-index.html

```diff
@@ -0,0 +1,200 @@
+<div class="block">Used to exclude default interceptors for a target class or for a lifecycle callback method, business method, timeout
+ method, or constructor of a target class.</div>
+</th>
+</tr>
+<tr class="rowColor" id="i5">
+<td class="colFirst"><a href="javax/interceptor/Interceptor.html" title="annotation in javax.interceptor">Interceptor</a></td>
+<th class="colLast" scope="row">
+<div class="block">Specifies that a class is an interceptor.</div>
+</th>
+</tr>
+<tr class="altColor" id="i6">
+<td class="colFirst"><a href="javax/interceptor/Interceptor.Priority.html" title="class in javax.interceptor">Interceptor.Priority</a></td>
+<th class="colLast" scope="row">
+<div class="block">
+ Priorities that define the order in which interceptors are invoked.</div>
+</th>
+</tr>
+<tr class="rowColor" id="i7">
+<td class="colFirst"><a href="javax/interceptor/InterceptorBinding.html" title="annotation in javax.interceptor">InterceptorBinding</a></td>
+<th class="colLast" scope="row">
+<div class="block">Specifies that an annotation type is an interceptor binding type.</div>
+</th>
+</tr>
+<tr class="altColor" id="i8">
+<td class="colFirst"><a href="javax/interceptor/Interceptors.html" title="annotation in javax.interceptor">Interceptors</a></td>
+<th class="colLast" scope="row">
+<div class="block">Declares an ordered list of interceptors for a target class, or for a method or a constructor of a target class.</div>
+</th>
+</tr>
+<tr class="rowColor" id="i9">
+<td class="colFirst"><a href="javax/interceptor/InvocationContext.html" title="interface in javax.interceptor">InvocationContext</a></td>
+<th class="colLast" scope="row">
+<div class="block">Exposes contextual information about the intercepted invocation and operations that enable interceptor methods to
+ control the behavior of the invocation chain.</div>
+</th>
+</tr>
+</tbody>
+</table>
+</div>
+</div>
+</li>
+</ul>
+</div>
+</main>
+<footer role="contentinfo">
+<nav role="navigation">
+<!-- ======= START OF BOTTOM NAVBAR ====== -->
+<div class="bottomNav"><a id="navbar.bottom">
+<!--   -->
+</a>
+<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
+<a id="navbar.bottom.firstrow">
+<!--   -->
+</a>
+<ul class="navList" title="Navigation">
+<li><a href="javax/interceptor/package-summary.html">Package</a></li>
+<li>Class</li>
+<li>Use</li>
+<li><a href="javax/interceptor/package-tree.html">Tree</a></li>
+<li><a href="deprecated-list.html">Deprecated</a></li>
+<li><a href="index-all.html">Index</a></li>
+<li><a href="help-doc.html">Help</a></li>
+</ul>
+</div>
+<a id="skip.navbar.bottom">
+<!--   -->
+</a>
+<!-- ======== END OF BOTTOM NAVBAR ======= -->
+</nav>
+<p class="legalCopy"><small>Copyright &#169; 1996-2013,     <a href="http://www.oracle.com">Oracle</a>     and/or its affiliates. All Rights Reserved.     Use is subject to     <a href="./doc-files/speclicense.html" target="_top">license terms</a>.</small></p>
```

Comment:
@arjantijms, the footer should refer to the EFSL and there should be no mention of Oracle as the copyright holder.

Reply:
You’re right, thx! I’m going through a series of all specs where this needs to be updated.